arm/Microsoft.Batch/batchAccounts/deploy.bicep

```
@@ -32,6 +65,30 @@ param lock string = 'NotSpecified'
 @description('Optional. Tags of the resource.')
 param tags object = {}
 
+@allowed([
+  'AAD'
+  'SharedKey'
+  'TaskAuthenticationToken'
+])
+@description('Optional. List of allowed authentication modes for the Batch account that can be used to authenticate with the data plane.')
+param allowedAuthenticationModes array = []
+
+@allowed([
+  'Microsoft.Batch'
+  'Microsoft.KeyVault'
+])
+@description('Otional. Type of the key source.')
```

```suggestion
@description('Optional. Type of the key source.')
```
Please regenerate readme